### PR TITLE
Amp examples

### PR DESCRIPTION
**Pull request by tuck1s**
Changed to use  https://playground.amp.dev/ for examples. The code is embedded in the (long) URLs rather than relying on a codepen account.
Plus other tidy-ups.
See ESCSP-4427

### CODE REVIEW

File: support/user-guide/amp-for-email.md [hunk 1]
```diff
@@ -118,16 +118,21 @@ Example:
...
...m1pdHMgLS0+CiAgICAgIDxmb3JtIGFjdGlvbi14aHI9Imh0dHBzOi8vYW1wYnlleGFtcGxlLmNvbS9zYW1wbGVzX3RlbXBsYXRlcy9wb2xsL3N1Ym1pdCIgbWV0aG9kPSJwb3N0Ij4KICAgICAgICA8ZGl2IGlkPSJmb3JtLWNvbnRlbnQiPgogICAgICAgICAgPGgxPlRoYW5rcyBmb3Igc3Vic2NyaWJpbmcgdG8gU3BhcmtQb3N0IEVtYWlscyE8L2gxPgogICAgICAgICAgPHA+V2hhdCBraW5kIG9mIGVtYWlscyBzaG91bGQgd2Ugc2VuZCB5b3U/PC9wPgogICAgICAgICAgPGFtcC1zZWxlY3RvciBuYW1lPSJzdWJzY3JpcHRpb25zIiBtdWx0aXBsZT4KICAgICAgICAgICAgPGRpdiBvcHRpb249ImFjY291bnQtbWV0cmljcyIgc2VsZWN0ZWQ+V2Vla2x5IEFjY291bnQgTWV0cmljczwvZGl2PgogICAgICAgICAgICA8ZGl2IG9wdGlvbj0iZGFpbHktcmV2aWV3cyIgc2VsZWN0ZWQ+RGFpbHkgRGVsaXZlcmFiaWxpdHkgUmVwb3J0PC9kaXY+CiAgICAgICAgICAgIDxkaXYgb3B0aW9uPSJuZXdzbGV0dGVyIiBzZWxlY3RlZD5CZXRhIFRlc3RlciBOZXdzbGV0dGVyPC9kaXY+CiAgICAgICAgICAgIDxkaXYgb3B0aW9uPSJwcm9kdWN0LXVwZGF0ZXMiIHNlbGVjdGVkPlByb2R1Y3QgVGVhbSBVcGRhdGVzPC9kaXY+CiAgICAgICAgICA8L2FtcC1zZWxlY3Rvcj4KICAgICAgICAgIDxpbnB1dCB0eXBlPSJzdWJtaXQiIHZhbHVlPSJVcGRhdGUiIGNsYXNzPSJ1cGRhdGVCdXR0b24iPgogICAgICAgIDwvZGl2PgogICAgICAgIDxkaXYgc3VibWl0LXN1Y2Nlc3M+CiAgICAgICAgICA8aDE+R290IGl0ISBXZSB3aWxsIHVwZGF0ZSB5b3VyIHByZWZlcmVuY2VzIHdpdGhpbiAyNCBob3VyczwvaDE+CiAgICAgICAgPC9kaXY+CiAgICAgIDwvZm9ybT4KICAgIDwvZGl2PgogIDwvYm9keT4KPC9odG1sPg==)
```
Comment (bkemper) [inline support/user-guide/amp-for-email.md:124]: There is an error in this example.  The form submit request fails.  We should probably move completely away from "https://ampbyexample.com".

Screen Shot 2021-01-08 at 2 17 29 PM

Comment (tuck1s) [reply support/user-guide/amp-for-email.md:124]: Yes, had not noticed that myself. Looks like that domain has been deprecated and content moved - see https://blog.amp.dev/2019/04/02/ampproject-org-moves-to-amp-dev/. I'll try to find out the new URL to use.

Comment (tuck1s) [reply support/user-guide/amp-for-email.md:124]: This example seems close: https://amp.dev/documentation/components/amp-form/?format=email

Comment (tuck1s) [reply support/user-guide/amp-for-email.md:124]: I can make the example work using
```
      <form action-xhr="/documentation/examples/api/submit-form-input-text-xhr" method="post">

```

This hits an endpoint on the amp.dev site

Comment (bkemper) [reply support/user-guide/amp-for-email.md]: > The relative URL '/documentation/examples/api/submit-form-input-text-xhr' for attribute 'action-xhr' in tag 'form' is disallowed.

It looks like the playground validation doesn't like the relative link, but prepending "https://amp.dev/" works!

Screen Shot 2021-01-13 at 11 26 53 PM

Comment (tuck1s) [reply support/user-guide/amp-for-email.md]: Good catch. Will change it.

**Review by bkemper — APPROVED** [review]
LGTM!  Merge when ready and I'll manually deploy ASAP.

**Pull request merged**